spec/controllers/idv/in_person/address_controller_spec.rb

```diff
@@ -68,12 +81,15 @@
       expect(response).to render_template :show
     end
 
-    it 'redirects to ssn page when address1 present' do
-      subject.user_session['idv/in_person'][:pii_from_user][:address1] = '123 Main St'
-
-      get :show
+    context 'when address1 present' do
```

Comment:
nit: What do you think about changing context here to be 'when residential address matches state-issued id address' (or something along those line)?

Reply:
Gina and I chatted via slack and decided to keep this context block.  It represents a scenario where the residential address and state id address **don't** match.